src/js/hca-rjsf/config/form.js

```diff
@@ -24,6 +46,91 @@ const formConfig = {
           schema: {}
         }
       }
+    },
+    militaryService: {
+      title: 'Military service',
+      pages: {
+        serviceInformation: {
+          path: 'military-service/service-information',
+          title: 'Service periods',
+          uiSchema: {
+            lastServiceBranch: {
+              'ui:title': 'Last branch of service',
+              'ui:options': {
+                labels: lastServiceBranchLabels
+              }
+            },
+            // TODO: this should really be a dateRange, but that requires a backend schema change. For now
+            // leaving them as dates, but should change these to get the proper dateRange validation
+            lastEntryDate: currentOrPastDate.uiSchema('Start of service period'),
```

Comment:
Does "they do not" mean they *should* not, or they just didn't previously, but it would be fine if they did? @annekainicUSDS maybe you have more bg information on hca requirements